server/db/svarog-11.psql

```diff
@@ -0,0 +1,38 @@
+-- This schema version changes the config columns from varchar to jsonb.
+
+ALTER TABLE observations ALTER COLUMN config TYPE jsonb USING json_build_object('text',config);
+ALTER TABLE stations ALTER COLUMN config TYPE jsonb USING json_build_object('text',config);
+
+-- Now we need to migrate existing data. This is really specific to station 1.
+-- Let's add protocol transmission. So far we were able to receive APT and LRPT.
+UPDATE observations SET config = config || '{"protocol": "APT" }' WHERE sat_id IN (25338, 28654, 33591) AND station_id = 1;
+UPDATE observations SET config = config || '{"protocol": "LRPT" }' WHERE sat_id IN (40069, 44387) AND station_id = 1;
+
+-- All transmissions received so far were done using crossed dipole
+UPDATE observations SET config = config || '{"antenna": "WiMo TA-1", "antenna-type": "crossed dipole" }' WHERE station_id = 1;
+
+-- Add frequencies. We haven't done any Doppler magic, so everything was received on base frequencies.
+UPDATE observations SET config = config || '{"frequency": 137.62e6 }' WHERE sat_id = 25338 AND station_id = 1; -- NOAA 15
```

Comment:
Frequency is in Hz or MHz, finally?

Reply:
Hz. I've updated the code as you suggested.